app/lib/cognito.server.ts

```diff
@@ -18,15 +18,26 @@ import {
   CreateGroupCommand,
   DeleteGroupCommand,
   GetGroupCommand,
+  ListUsersCommand,
   UpdateGroupCommand,
   paginateAdminListGroupsForUser,
   paginateListGroups,
+  paginateListUsers,
   paginateListUsersInGroup,
 } from '@aws-sdk/client-cognito-identity-provider'
 
 export const cognito = new CognitoIdentityProviderClient({})
 const UserPoolId = process.env.COGNITO_USER_POOL_ID
 
+export interface UserData {
+  email: string
+  sub?: string
+  name?: string
+  affiliation?: string
+  receive?: boolean
+  submit?: boolean
+}
```

Comment:
This should not go in `cognito.server.ts`. This file contains utilities for dealing with Cognito and should not have any details about our own data model.